### PR TITLE
Introduce new release (stable) version 2019-01-01 (no APIs have been added or updated)

### PR DESCRIPTION
**Pull request by shbha**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by shbha** [comment]
is there an action item here for me?

File: specification/consumption/resource-manager/readme.md [hunk 1]
```diff
@@ -336,3 +347,16 @@ regenerate-manager: true
 generate-interface: true
 ```
 
+### Tag: package-2019-01 and java
```
Comment (veronicagg) [inline specification/consumption/resource-manager/readme.md]: please take a look at the readme file for each of the other language SDKs located in the same folder as readme.md and update those too. Java setting are included in this file, but the others have been extracted out.

Comment (veronicagg) [reply specification/consumption/resource-manager/readme.md]: Please check that all languages are generating SDKs for the PR, and that the generated code is resulting as expected (you should see links to PRs for each language posted as messages in this PR).

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json [hunk 2]
```diff
@@ -91,7 +91,259 @@
...
+          "UsageDetailsList": {
+            "$ref": "./examples/UsageDetailsListByBillingAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "name": "$expand",
```
Comment (veronicagg) [inline specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json]: Please take a look at https://github.com/Azure/autorest/blob/6d41b4e174626aea95f76d68f7ac4d082e291aa4/docs/extensions/readme.md#x-ms-odata and evaluate if it makes sense for some of your operations.

**Review by veronicagg — CHANGES_REQUESTED** [review]
Please take a look at the comments inline. 
Is the new version just introducing some more operations?
Please take a look at the warnings at https://portal.azure-devex-tools.com/app/branch/shbha/azure-rest-api-specs/master/linter and evaluate whether you'd like to address any of those. 
I have tagged this PR for the ARM team to sign-off on your changes.
Thanks!

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json (hunk 2 above)
Comment (shbha) [reply specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json]: Thanks for your feedback. I'll consider this for future PRs.

**Comment by shbha** [comment]
> @shbha are you copying from an existing api-version version, it appears that you are based on the commits, Did you look at using the [OpenAPI Hub](http://aka.ms/openapihub) to make this change? it makes it easier for us to review, as we can see different commits with the changes. It also makes a lot of the changes regarding api-version for you automatically, like creating a new api-version folder with all the files copied, updating the api-versions, updating the readme.

*I wanted to use OpenAPI Hub, but I wasn't able to go beyond joining the Microsoft Opensource group. Each time I click on the link it asks me to join the group (which I'v already joined) but never takes me to the OpenAPI Hub itself. 
How do you suggest we proceed? I did notice the commits, but I hope for this PR we can go by the commit description that explicitly states that a new 2019-01-01 folder is being introduced? Thanks

**Comment by shbha** [comment]
@veronicagg - There are no new operations being introduced. The commit you've shared is from last year, thats already merged in. Please refer to changes after the new folder (2019-01-01) has been created - [commit](https://github.com/Azure/azure-rest-api-specs/pull/5110/commits/0a93c4c708d208fe78e01ae4cbe25f0187a08ff6)

**Comment by veronicagg** [comment]
Talked to @shbha offline, the intention is to release a new api-version with the same content and APIs as the existing, this PR is getting replaced by https://github.com/Azure/azure-rest-api-specs/pull/5139

**Pull request closed without merging**